api/src/opentrons/protocol_api/load_info.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from typing import Optional
+from opentrons_shared_data.labware.dev_types import LabwareDefinition
+
+from opentrons.types import Mount, DeckSlotName, DeckLocation
+
+
+@dataclass(frozen=True)
+class LabwareLoadInfo:
+    """For Opentrons internal use only.
+
+    :meta private:
+
+    Information about a successful labware load.
+
+    This is a separate class from the main user-facing `Labware` class
+    because this is easier to construct in unit tests.
+    """
+
+    labware_definition: LabwareDefinition
+    # todo(mm, 2021-10-11): Namespace, load name, and version can be derived from the
+    # definition. Should they be removed from here?
+    labware_namespace: str
+    labware_load_name: str
+    labware_version: int
+    deck_slot: DeckSlotName
+    on_module: Optional[bool] = False
```

Comment:
We don't need the `Optional` here thanks to the default value

```suggestion
    on_module: bool = False
```